### DISCUSSION TITLE
./gradlew core:zipRiruRelease

### DISCUSSION THREAD

**Discussion by nkbai** [LSPosed/LSPosed#1736] [Q&A]
Configuration on demand is an incubating feature.

> Configure project :core
WARNING:The option setting 'android.experimental.enableNewResourceShrinker.preciseShrinking=true' is experimental.
The current default is 'false'.
WARNING:The option setting 'android.enableAppCompileTimeRClass=true' is experimental.
The current default is 'false'.

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':core:prepareMagiskFilesRiruRelease'.
> A problem occurred configuring project ':daemon'.
   > Extension with name 'android' does not exist. Currently registered extension names: [ext]

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

**Comment by yujincheng08** [comment]
Extension with name 'android' does not exist

**Comment by nkbai** [comment]
where can i found the step by step  build instruction of LSPOSED?  

I don't know where to get this `android` extension。

**Reply by yujincheng08** [reply]
https://github.com/LSPosed/LSPosed/blob/4b9a4f83161d6a4a5d2106cd0217fc86919827e3/.github/workflows/core.yml#L79

**Reply by yujincheng08** [reply]
android plugin: https://developer.android.com/studio/releases/gradle-plugin

it will be installed automatically by gradle.

your error is because gradle is not working properly.

check your internet connection.

**Reply by nkbai** [reply]
thanks！ it works by `./gradlew zipAll `, but  `./gradlew core:zipRiruRelease` still has the same problem.
